### PR TITLE
release: 4.5.0

### LINKED ISSUE #5307: [Chore]: release 4.5.0

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR is to generate a new release of `code-server` at `v4.5.0`

## Screenshot

image

## TODOs

Follow "Publishing a release" steps in `ci/README.md`

- [ ] publish release and merge PR
- [ ] update the AUR package
- [x] create issue to fix e2e test

Fixes #5307

> **Note**
> GitHub changed some validation around tags/branches. We need the branch to be different than the tag i.e. v4.5.0

We had to do this a little differently. @Emyrk and I agreed we can merge this PR, delete the branch and then publish the release.

### CODE REVIEW

File: test/e2e/openHelpAbout.test.ts [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
     // Open using the menu.
     await codeServerPage.navigateMenus(["Help", "About"])
 
-    const isDevMode = process.env.VSCODE_DEV === "1"
-
     // Look for code-server info div.
-    const element = await codeServerPage.page.waitForSelector(
-      `div[role="dialog"] >> text=code-server: ${isDevMode ? "Unknown" : "v" + version}`,
-    )
+    const element = await codeServerPage.page.waitForSelector(`div[role="dialog"] >> text=code-server`)
```
Comment (jsjoeio) [inline test/e2e/openHelpAbout.test.ts:9]: I spent half a day trying to debug this. The dialog shows the version as expected but running locally vs in CI results in different outcomes. I will fix this next sprint: https://github.com/coder/code-server/issues/5312

File: docs/MAINTAINING.md [hunk 2]
```diff
@@ -164,13 +164,14 @@ If you're the current release manager, follow these steps:
...
-1. Create a new branch called `v0.0.0` (replace 0s with actual version aka v4.4.0)
+1. Create a new branch called `v0.0.0` (replace 0s with actual version aka v4.5.0)
 1. Run `yarn release:prep` and type in the new version (e.g., `3.8.1`)
 1. GitHub Actions will generate the `npm-package`, `release-packages` and
    `release-images` artifacts. You do not have to wait for this step to complete
    before proceeding.
 1. Run `yarn release:github-draft` to create a GitHub draft release from the
-   template with the updated version.
+   template with the updated version. Make sure to update the `CHANGELOG.md`.
+1. Bump chart version in `Chart.yaml`.
```
Comment (Kira-Pilot) [inline docs/MAINTAINING.md:174]: lot of step 1s here 😄

**Review by Kira-Pilot — APPROVED** [review]

File: docs/MAINTAINING.md (hunk 2 above)
Comment (jsjoeio) [reply docs/MAINTAINING.md:174]: Ah yes, that's intentional! That way if you re-order, you don't have to go renumber them.

 🖼️ [rendered on GitHub](https://github.com/coder/code-server/blob/main/docs/MAINTAINING.md#publishing-a-release)

[Source](https://github.github.com/gfm/#list-items)

Comment (Kira-Pilot) [reply docs/MAINTAINING.md:174]: TIL! Thanks for the link!

**Pull request merged**